templates/repo/issue/list.tmpl

```diff
@@ -162,36 +162,38 @@
 		</div>
 
 		<div class="issue list">
-			{{range .Issues}}
-				{{ $timeStr:= TimeSince .Created $.Lang }}
+			{{range $index, $issue := .Issues}}
```

Comment:
Is `$index` ever used? Similarly for the other template.

Reply:
Seems no needed after my second push.